addons/website/tests/test_ui.py

```diff
@@ -61,24 +61,24 @@ class TestUiHtmlEditor(odoo.tests.HttpCase):
     def test_html_editor_multiple_templates(self):
         Website = self.env['website']
         View = self.env['ir.ui.view']
-        generic_aboutus = Website.viewref('website.aboutus')
+        generic_homepage = Website.viewref('website.homepage')
```

Comment:
wrong exemple

hompage is bootsrapped during the creation of the website, so it is not generic